src/main/java/io/apiman/cli/command/plan/model/PlanVersion.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Jean-Charles Quantin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apiman.cli.command.plan.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Models an Plan version.
+ *
+ * @author Jean-Charles Quantin {@literal <jeancharles.quantin@gmail.com>}
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PlanVersion {
+    @JsonProperty
+    private String version;
+
+    /**
+     * Never clone a previous version when creating a new version.
+     */
+    @JsonProperty
+    final private boolean clone = false;
```

Comment:
Yes it works like the POST of api's version (@POST("/organizations/{orgName}/plans/{planName}/versions")
The clone's mecanism is the same as Api one.